deluge/plugins/AutoAdd/deluge/plugins/autoadd/core.py

```diff
@@ -253,7 +253,7 @@ def update_watchdir(self, watchdir_id):
                 continue
             else:
                 ext = os.path.splitext(filename)[1]
-                if ext == ".torrent":
+                if ext == ".torrent" or ".TORRENT":
```

Comment:
Ah good point. I do not code in python, I thought this was shorthand. I am going to add what aresch suggested and just make the extension lowercase.